<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 219, 189);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Mira fijamente al centro de los puntos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="calibrationDotBase">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>345</y>
     <width>59</width>
     <height>59</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="calibrationDotCross_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0.48003 rgba(255, 255, 255, 0), stop:0.48154 rgba(0, 0, 0, 255), stop:0.52001 rgba(0, 0, 0, 255), stop:0.52112 rgba(255, 255, 255, 0))
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="calibrationDotBorder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.95, fx:0.5, fy:0.5, stop:0.392962 rgba(0, 0, 0, 0), stop:0.394428 rgba(0, 0, 0, 255), stop:0.470674 rgba(0, 0, 0, 255), stop:0.473607 rgba(0, 0, 0, 0));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="calibrationDotCenter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.2, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.519685 rgba(0, 0, 0, 255), stop:0.524752 rgba(255, 255, 255, 0), stop:0.99802 rgba(255, 255, 255, 0));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="calibrationDotBg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.85, fx:0.5, fy:0.5, stop:0 rgba(255, 0, 0, 255), stop:0.519685 rgba(255, 0, 0, 255), stop:0.524752 rgba(255, 255, 255, 0), stop:0.99802 rgba(255, 255, 255, 0));
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="calibrationDotCross_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:0, y2:1, stop:0.48003 rgba(255, 255, 255, 0), stop:0.48154 rgba(0, 0, 0, 255), stop:0.52001 rgba(0, 0, 0, 255), stop:0.52112 rgba(255, 255, 255, 0))
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>calibrationDotBg</zorder>
   <zorder>calibrationDotBorder</zorder>
   <zorder>calibrationDotCenter</zorder>
   <zorder>calibrationDotCross_1</zorder>
   <zorder>calibrationDotCross_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
